Source/GitLab/gitlab-utils.env

```diff
@@ -63,14 +63,14 @@ _t4dGitLabSafetyCheck(){
 
     ## Gitlab Token #####################################
 
-    [[ "${GITLAB_TOKEN}" == "" ]] && (>&2 echo "${plred}ERROR 320${pstd} in bb8 : GITLAB_TOKEN variable is empty") && return 320
+    [[ "${GITLAB_TOKEN}" == "" ]] && (>&2 echo "${plred}ERROR 320${pstd} in ${1:-bb8} : GITLAB_TOKEN variable is empty") && return 320
 
     local _OuputjsonFile="$T4D_PROJECT_LOG_PATH/gitlab/_t4dGitLabGetMainPage.json"
     if [[ "$(_t4dGitLabGetMainPage $_OuputjsonFile 2>&1 | grep 'ERROR')" != "" ]]; then
-        _t4dPromptError "320 in bb8" "Can't Reach GitLab - https://$_t4dGitLabURL" && return 320
+        _t4dPromptError "320 in ${1:-bb8}" "Can't Reach GitLab - https://$_t4dGitLabURL" && return 320
     fi
     if [[ "$(_t4dGitLabCheckLogIn)" != 'null' ]]; then
-        _t4dPromptError "320 in bb8" "$(_t4dGitLabCheckLogIn)" && return 320
```

Comment:
what is the usage of this ?

Reply:
Kept previous usage while allowing to reuse the function from another module with "valid" error message (correct module name). Usefull for wci with gitlab : status check before ci api queries